.github/workflows/release.yaml

```diff
@@ -19,21 +19,25 @@ jobs:
       - uses: actions/checkout@v2
       - run: git fetch --prune --unshallow
       - uses: actions/setup-go@v2
-
       - name: release dry-run
-        uses: goreleaser/goreleaser-action@v1
-        with:
-          args: --skip-validate --skip-publish
-          version: v0.133.0
-
-      - if: startsWith(github.ref,'refs/tags/v') && github.repository == 'ovrclk/akash'
-        name: docker login
-        run: echo '${{secrets.DOCKERHUB_PASSWORD}}' | docker login --username '${{secrets.DOCKERHUB_USERNAME}}' --password-stdin
+        run: |-
+          docker run \
+          --priviledged \
+          --env MAINNET=${MAINNET} \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          -v `pwd`:/go/src/github.com/ovrclk/akash \
+          goreng/golang-cross:latest \
```

Comment:
Can you pick out the version which `latest` points to? `latest` is always asking for problems later.